### PR TITLE
Updating TSA Updater to include icu data + fix mishandled dates

### PR DESCRIPTION
**Pull request by ghop02**

### CODE REVIEW

File: scripts/update_texas_fips_hospitalizations.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ class Fields(common_fields.GetByValueMixin, common_fields.FieldNameAndCommonFiel
     DATE = "date", CommonFields.DATE
     FIPS = CommonFields.FIPS, CommonFields.FIPS
     CURRENT_HOSPITALIZED = CommonFields.CURRENT_HOSPITALIZED, CommonFields.CURRENT_HOSPITALIZED
+    CURRENT_ICU = CommonFields.CURRENT_ICU, CommonFields.CURRENT_ICU
```
Comment (TomGoBravo) [inline scripts/update_texas_fips_hospitalizations.py:23]: I missed this earlier, but the first argument is expected to be a literal string unless the file was written by CAN code using CommonFields.

File: scripts/update_texas_fips_hospitalizations.py [hunk 2]
```diff
@@ -20,6 +20,7 @@ class Fields(common_fields.GetByValueMixin, common_fields.FieldNameAndCommonFiel
...
```
Comment (TomGoBravo) [inline scripts/update_texas_fips_hospitalizations.py:24]: Add
TSA_REGION_ID = "TSA ID", None
and use it, similar to update_texas_tsa_hospitalizations.py

File: scripts/update_texas_fips_hospitalizations.py (hunk 1 above)
Comment (ghop02) [reply scripts/update_texas_fips_hospitalizations.py:23]: I think it was? The texas tsa file is written by can code using that key

File: scripts/update_texas_fips_hospitalizations.py (hunk 2 above)
Comment (ghop02) [reply scripts/update_texas_fips_hospitalizations.py:24]: Ah these are supposed to be the input fields, correct?

Comment (TomGoBravo) [reply scripts/update_texas_fips_hospitalizations.py:26]: Yes. ENUM = "input field", CommonField.....

**Pull request merged**